src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -20,8 +21,7 @@ Napi::Function ModelWrap::get_class(Napi::Env env) {
                         InstanceMethod("output", &ModelWrap::get_output),
                         InstanceMethod("input", &ModelWrap::get_input),
                         InstanceMethod("isDynamic", &ModelWrap::is_dynamic),
-                        InstanceMethod("setFriendlyName", &ModelWrap::set_friendly_name),
-                        InstanceMethod("getFriendlyName", &ModelWrap::get_friendly_name),
```

Comment:
Why these methods were removed?

Reply:
Sorry my bad there was a merge conflict I guess,On it!